### ISSUE TITLE
Is there a way to forward just one device connection from usbmuxd

### ISSUE THREAD

**Issue by harrain** [libimobiledevice/libusbmuxd#105]
When I use socat to forward /var/run/usbmuxd to tcp port:

socat TCP-LISTEN:1234,reuseaddr,fork UNIX-CONNECT:/var/run/usbmuxd

It will transfer all ios devices. Is there and way to transfer just one device ???

I want to share my ios device via internet, but I just want to share only 1 device for 1 client.

**Comment by mexmer** [comment]
no, because you forwarding connection to USBMUXD, not to device.

usbmuxd forwards individual TCP sockets from device, but there is only 1 socket open on device all time, and that is lockdown service, other sockets will open, when you start service on device, and it will open at random port. so you will need to create new forward each time service socket is opened on device.

**Comment by harrain** [comment]
So how to create new forward from LOCKDOWN service？appreciate

**Comment by mexmer** [comment]
you need to write your own service, that will do it.
